lib/teleterm/daemon/daemon.go

```diff
@@ -322,21 +323,36 @@ func (s *Service) SetGatewayLocalPort(gatewayURI, localPort string) (*gateway.Ga
 	return newGateway, nil
 }
 
-// ListServers returns cluster servers
-func (s *Service) ListServers(ctx context.Context, clusterURI string) ([]clusters.Server, error) {
+// GetAllServers returns a full list of nodes without pagination or sorting.
```

Comment:
Most of these APIs use the term servers, but in the godoc you refer to nodes. Are you using these terms interchangeably or was the switch to nodes intentional?

Reply:
Accidental use of nodes. changing to servers, thanks. Had "nodes" in my head after reviewing the web api a lot during this PR